tracing-subscriber/src/fmt/format/pretty.rs

```diff
@@ -219,6 +234,49 @@ impl<'writer> FormatFields<'writer> for Pretty {
     }
 }
 
+// === impl PrettyFields ===
+
+/// An excessively pretty, human-readable [`MakeVisitor`] implementation.
+///
+/// [`MakeVisitor`]: crate::field::MakeVisitor
+#[derive(Debug)]
+pub struct PrettyFields {
+    ansi: bool,
+    // reserve the ability to add fields to this without causing a breaking
+    // change in the future.
+    _private: (),
```

Comment:
i don't think we need this anymore, since we already have a private field (`ansi`).

Reply:
Fixed in 779f53b :sparkles: